src/nupic/algorithms/Anomaly.cpp

```diff
@@ -109,6 +108,8 @@ Real32 Anomaly::compute(
   {
     score = (score >= binaryThreshold_) ? 1.0 : 0.0;
   }
+  // round result (to avoid near zero rounding errors)
+  score = roundf((score*PRECISION)/PRECISION);
```

Comment:
thanks @scottpurdy ! Fixed that and works fine. (should still remove for separate PR?)